asreview/webapp/task_manager/models.py

```diff
@@ -0,0 +1,16 @@
+from sqlalchemy import Boolean
+from sqlalchemy import Column
+from sqlalchemy import Integer
+from sqlalchemy import String
+from sqlalchemy.ext.declarative import declarative_base
+
+Base = declarative_base()
+
+
+class ProjectQueueModel(Base):
+    """Queue model"""
+
+    __tablename__ = "queue"
+    id = Column(Integer, primary_key=True)
+    project_id = Column(String(250), nullable=False, unique=True)
+    simulation = Column(Boolean, nullable=False, default=False)
```

Comment:
I foresee that we will use this queue for more tasks in the future (full text parsing or LLMs). It might be useful to use the project "modes" we are currently using for other things. 

```suggestion
    mode = Column(String(100), nullable=False, default="oracle")
```